### ISSUE TITLE
URL behind "My Work Items" does not show only my work items but all

### ISSUE THREAD

**Issue by kwk** [fabric8io/fabric8-planner#1867]
I'm not sure if this is a bug in platform or planner but in the Analyze section there's this box:

<img width="674" alt="bildschirmfoto 2017-05-09 um 20 15 52" src="https://cloud.githubusercontent.com/assets/193408/25865793/75bb79ac-34f4-11e7-8cbe-6d1aae032436.png">

The URL behind the "My Work Items" link is `https://openshift.io/kkleine/test_2/plan`but I think it should be `https://openshift.io/kkleine/test_2/plan?assignee=kkleine%20(me)` in order to show only my work items and not the ones that are generally in the space.

**Comment by catrobson** [comment]
Agreed that this should link to a filtered list of work items assigned to me.
